modules/tensor_mechanics/test/tests/shell/static/tests

```diff
@@ -0,0 +1,45 @@
+[Tests]
+  design = "ShellElements.md"
+  issues = "#14280"
+  [./beam_bending]
+    type = 'Exodiff'
+    input = 'beam_bending_moment_AD.i'
+    exodiff = 'beam_bending_moment_AD_out.e'
+
+    requirement = "The mechanics system shall accurately compute the deflection of a cantilever beam when it is modeled using shell elements."
+  [../]
+  [./large_beam_bending]
+    type = 'Exodiff'
+    input = 'large_strain_m_40_AD.i'
+    exodiff = 'large_strain_m_40_AD_out.e'
+
+    requirement = "The mechanics system shall accurately compute the deflection of a cantilever beam when it is modeled using shell elements under large strain and rotations are included."
+  [../]
+  [./beam_bending_jacobian]
+    type = 'PetscJacobianTester'
+    input = 'beam_bending_moment_AD.i'
+
+    requirement = "The mechanics system shall accurately compute the Jacobian for a small strain quasi-static shell element."
+  [../]
+  [./large_beam_bending_jacobian]
+    type = 'PetscJacobianTester'
+    input = 'beam_bending_moment_AD.i'
```

Comment:
It looks like you're just running the same input file as above without any `cli_args`, so it looks like an identical test. Probably not what you meant to do based on the different requirements?